ansible/roles/cli/tasks/download_openwhisk_cli.yml

```diff
@@ -69,3 +71,9 @@
     chdir: "{{ cli.nginxdir }}/{{ os }}/{{ arc }}/"
   when: os == "linux"
 
+- name: "modify the mode of Nginx cli directory for ({{ os }}) {{ arc }}"
```

Comment:
remove this since you are already setting permission to `0777` before on the same directory

Reply:
It is good to keep the folder 0777, which means anyone can do anything. We set it to 0777 in order to upload the binaries. After it is done, do we still keep the 0777?